prow/def.bzl

```diff
@@ -95,11 +103,23 @@ def target(cmd):
 #     "gcr.io/k8s-prow/ghproxy:20180203-deadbeef": "//ghproxy:image",
 #     "gcr.io/k8s-prow/ghproxy:latest": "//ghproxy:image",
 #     "gcr.io/k8s-prow/ghproxy:latest-fejta": "//ghproxy:image",
+#     "gcr.io/k8s-prow-edge/hook:20180203-deadbeef": "//prow/cmd/hook:image",
+#     "gcr.io/k8s-prow-edge/hook:latest": "//prow/cmd/hook:image",
+#     "gcr.io/k8s-prow-edge/hook:latest-fejta": "//prow/cmd/hook:image",
+#     "gcr.io/k8s-prow-edge/plank:20180203-deadbeef": "//prow/cmd/plank:image",
+#     "gcr.io/k8s-prow-edge/plank:latest": "//prow/cmd/plank:image",
+#     "gcr.io/k8s-prow-edge/plank:latest-fejta": "//prow/cmd/plank:image",
+#     "gcr.io/k8s-prow-edge/ghproxy:20180203-deadbeef": "//ghproxy:image",
+#     "gcr.io/k8s-prow-edge/ghproxy:latest": "//ghproxy:image",
+#     "gcr.io/k8s-prow-edge/ghproxy:latest-fejta": "//ghproxy:image",
 #   }
 def tags(cmds, targets):
     # Create :YYYYmmdd-commitish :latest :latest-USER tags
     cmd_targets = {prefix(cmd): target(cmd) for cmd in cmds}
     cmd_targets.update({prefix(p): t for (p, t) in targets.items()})
+    if "{EDGE_PROW_REPO}":
```

Comment:
Don't you think this would be easier to understand if we set EDGE_PROW_REPO = "{EDGE_PROW_REPO}" on ~L66?

Reply:
/shrug
Seems about as readable to me, I was following the pattern used for `STABLE_PROW_REPO`. Updated though.